src/deck-components/index.ts

```diff
@@ -21,3 +21,10 @@ export * from './Tabs';
 export * from './TextField';
 export * from './Toggle';
 export * from './ToggleField';
+export * from './SteamClient';
+
+import {SteamClient} from './SteamClient'
+
+declare global {
+    var SteamClient: SteamClient;
+}
```

Comment:
Does does TypeScript handle it correctly if my current file does not import anything from `decky-frontend-lib`? Like, will the globals still be recognized?

I will try it out later, but maybe one of you knows already...

Reply:
I don't know, it was suggested by @AAGaming00

Reply:
Likely not, but I don't see why that's an issue.

Reply:
Yea I guess you always have the import-chain from the file where `definePlugin` was used, so besides the newline, LGTM